data/themes/darktable-elegant-grey.css

```diff
@@ -74,6 +74,9 @@
 @define-color field_hover_bg @grey_75;
 @define-color field_hover_fg @grey_35;
 
+/* Tooltips and contextual helpers */
+@define-color tooltip_bg_color @grey_35;
```

Comment:
this should inherit the default CSS colour, that is better suited with bauhaus widget (sliders). Better leave it untouched in child-themes I think.

Reply:
having dark background on tooltips in a clearer theme like grey one is not consistent to the UI. So I don't see how to change that background another way. And I really don't see how a dark tooltip background is better suited with bauhaus widget.

Reply:
right click on a slider with this change and you will see that the cursor border (triangle) has a different colour. This is because I haven't been able to dynamically change the sliders border color. I agree it's not consistent but it's less ugly until I find a way to change cursor border color with CSS rules.

Reply:
This line doesn't affect the cursor border color, even in popup mode. It's only the really_dark line that you also removed that impact this. In fact, the border color of the slider cursor remains dark when popup activated instead of using bauhaus_indicator_border setting. So this tooltip line doesn't need to inherit from darktable.css theme. To be sure, I change this line to set a red color. See above, that doesn't affect popup right clider cursor slider :
![Capture-20191110145742-306x456](https://user-images.githubusercontent.com/45535283/68545458-9f0f9900-03cd-11ea-9886-3f8a3d3b53c6.png)

Reply:
> This line doesn't affect the cursor border color, even in popup mode. It's only the really_dark line that you also removed that impact this.

Oops, my bad, you are right. Proceed then.